flake.nix

```diff
@@ -98,15 +98,26 @@
 
       checks = forAllSystems (
         system:
-        {
-          tarball = jobs.${system}.tarball;
-        }
+        { }
```

Comment:
This hunk is different from the original PR: https://github.com/NixOS/nixpkgs/pull/395520

It removes the tarball check from `x86_64-freebsd` systems because `rustc-wrapper` is marked as broken on that arch.